### PR TITLE
Add style sheets for full calendar

### PR DESCRIPTION
**Pull request by SuyashSalampuria**
!important has to be used at 2 places.

### CODE REVIEW

**Comment by ayushjn20** [comment]
@pradumangoyal @Tushar19varshney Approve this PR ASAP.

File: omniport/omniport/static/full-calendar-semantic-ui/semantic.fullCalendar.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  color: #000000;
+}
+
+.fc-button-primary:focus {
+  box-shadow: 0px 0px 0px red;
+}
+
+.fc-day-number {
+  color: #3a87ad;
+}
```
Comment (algomaster99) [inline omniport/omniport/static/full-calendar-semantic-ui/semantic.fullCalendar.css]: @SuyashSalampuria please add a new line here. That's all I can say :laughing:

Comment (pradumangoyal) [reply omniport/omniport/static/full-calendar-semantic-ui/semantic.fullCalendar.css:74]: @SuyashSalampuria reminder lets merge this after this change, other things seem fine to me. The corresponding PR on the frontend can be merged only after this PR.

Comment (SuyashSalampuria) [reply omniport/omniport/static/full-calendar-semantic-ui/semantic.fullCalendar.css]: Done @pradumangoyal

**Comment by algomaster99** [comment]
@pradumangoyal Should we merge this?

**Comment by pradumangoyal** [comment]
@algomaster99 Yeah, just make sure a squash and a meaningful commit message.

**Comment by algomaster99** [comment]
We might not need this pull request. All the changes can itself be done on `omniport-frontend`. Here is the link @SuyashSalampuria is supposed to follow.

**Comment by pradumangoyal** [comment]
This CSS is a customised one; we changed the full calendar so that it fits with the semantic-ui.

**Comment by algomaster99** [comment]
@pradumangoyal Yeah, I had this discussion with @SuyashSalampuria. We are thinking to create a stylesheet in frontend itself where we will override the styles.

**Comment by pradumangoyal** [comment]
I am unsure about using the full calendar css file from there node-modules. The reason being the css-loader we have used in the frontend that hashes the class names from the CSS file imported in the JS files. I think due to this reason only we created a copy of semantic CSS at the backend and included in the index.html.

**Comment by algomaster99** [comment]
@pradumangoyal I think webpack won't hash the css from node_modules. And as far as writing custom css is concerned, we can escape that by prefixing classes with `:global`. Let @SuyashSalampuria give this a try, if it doesn't workout, we can revert back to the current way and try to reduce the redundancies.

**Comment by SuyashSalampuria** [comment]
@pradumangoyal @algomaster99 Yes, that was the issue due to which I had to add the css files here
Is there no way we can use import '~@fullcalendar/core/main.css'; in the frontend
I will give a try as @algomaster99 recommended. I remember importing this after a yarn install was not working at all

**Comment by algomaster99** [comment]
> import '~@fullcalendar/core/main.css';

Why did you add the `~`? That is scss syntax.

**Pull request merged**